src/components/Profile/ProfileInfo/ProfileInfo.js

```diff
@@ -15,7 +17,7 @@ const ProfileInfo = (props) => {
         <img src={BACKGROUND_IMG} />
       </div>
       <div className={style.descriptionBlock}>
-        <img src={props.profile.photos.large} />
+        <img src={profile.photos.large} />
```

Comment:
Consider adding "alt" attribute with generic image description here in every such place in your app.
The string value for the attribute could be stored in core/constants.
This will fix "img elements must have an alt prop, either with meaningful text, or an empty string for decorative images" linting error.

Reply:
/fxd